programs/cleos/main.cpp

```diff
@@ -3175,7 +3176,6 @@ int main( int argc, char** argv ) {
       }
    });
 
-   auto getSchedule = get_schedule_subcommand{get};
```

Comment:
Will removal of `get_schedule_subcommand{get}`  eliminate the `get schedule` command entirely? It seems like maybe only the `auto getSchedule =` should be removed?

Reply:
Thanks, you are correct. Added `get_schedule_subcommand{get};` back.